### DISCUSSION TITLE
Is there a way to, when adding a new router device, tell all devices to "reconnect" to whatever happens to be the closest/strongest node at that time?

### DISCUSSION THREAD

**Discussion by MostHated** [Koenkk/zigbee2mqtt#7186] [General]
Hey there, 
    I was having an issue with a few devices here and there getting low signal and not reporting often enough, so I added another routing device in between to hopefully pick up some slack. I added the new device, gave it an hour or three, looked at the map and noticed that nothing changed. I was never quite sure if devices would or could change which routing device they were immediately connected to in favor of one with a higher signal quality. Since I didn't see that happen, it looks like it doesn't happen automatically, but does there happen to exist a way to either send out some sort of "reshuffle" command in which devices could recheck on demand what device has the strongest signal quality to it and if it is not currently using that routing device, switch over and use the new one instead? 

If there is no way to request this in bulk, is there a way to initiate this somehow through the interface per device, or am I just dreaming that this type of functionality even exists in the first place and I need to just take the devices with low signal, remove them completely and then readd them a new?

If none of these exist, if possible, it would be amazing if this *could* be done. Possibly even from the map view, select a node which has low signal to it's current router, hit a button such as "re-home" or "re-base" or whatever, then select the new routing device with a better signal quality and then the device would begin reporting through that router instead.

Thanks,
-MH

**Comment by platini76** [comment]
I am also interested...
